tests/suites/secrets_iaas/juju.sh

```diff
@@ -121,7 +121,15 @@ run_user_secrets() {
 	# revision 1 should be pruned.
 	# revision 2 is still been used by the app, so it should not be pruned.
 	# revision 3 is the latest revision, so it should not be pruned.
-	check_contains "$(juju --show-log show-secret $secret_uri --revisions | yq ".${secret_short_uri}.revisions | length")" '2'
+	attempt=0
+	until [[ $(juju --show-log show-secret $secret_uri --revisions | yq ".${secret_short_uri}.revisions | length") -eq 2 ]]; do
+		if [[ ${attempt} -ge 30 ]]; then
+			echo "Failed: expected revision 1 get pruned."
```

Comment:
to get pruned
or
to be pruned
like is used in another place

Reply:
get -> gets